ts/store/reducers/backendStatus.ts

```diff
@@ -207,12 +208,15 @@ export const isPremiumMessagesOptInOutEnabledSelector = createSelector(
 
 /**
  * return the remote config about CDC enabled/disabled
- * if there is no data, false is the default value -> (CDC disabled)
+ * if there is no data or the local Feature Flag is disabled,
+ * false is the default value -> (CDC disabled)
  */
 export const isCdcEnabledSelector = createSelector(
   backendStatusSelector,
   (backendStatus): boolean =>
-    backendStatus.map(bs => bs.config.cdc.enabled).toUndefined() ?? false
+    (cdcEnabled &&
+      backendStatus.map(bs => bs.config.cdc.enabled).toUndefined()) ??
+    false
```

Comment:
```suggestion
    cdcEnabled &&
    backendStatus.map(bs => bs.config.cdc.enabled).getOrElse(false)
```
to avoid an extra conversion from `none` -> `undefined`

Reply:
Thank you! Refactored here: [63f42bb](https://github.com/pagopa/io-app/pull/3911/commits/63f42bbe8086ef3ea307bb270a1a4d2e272175fc)